C23_WMDE_Desktop_DE_25

Sperling-0 opened the pull request:
- Based on the VAR of test 24
- VAR has a collapsible banner

Ticket: https://phabricator.wikimedia.org/T353639

Abban commented:
Works okay in IE11

gbirke commented on desktop_preact/components/BannerText.jsx:
```diff
@@ -3,39 +3,62 @@ import { h } from 'preact';
+					{ campaignDaySentence }
+					<strong>Wikipedia wird durch Spenden von durchschnittlich 22,81 € finanziert. Wenn alle, die das jetzt
+						lesen, einen kleinen Beitrag leisten, wäre unser Spendenziel bereits am heutigen { currentDayName }
+						erreicht. Millionen Menschen nutzen Wikipedia, aber 99 % der Leserinnen und Leser spenden nicht. </strong>
```
<img width="406" alt="Bildschirmfoto 2020-07-10 um 17 56 39" src="https://user-images.githubusercontent.com/223326/87174552-9f9f5d00-c2d7-11ea-97e2-d0a1ed0369f8.png">

gbirke commented on desktop_preact/components/BannerText.jsx:
```diff
@@ -3,39 +3,62 @@ import { h } from 'preact';
+					</li>
+					<li>
+						<strong>Millionen Menschen nutzen Wikipedia, aber 99 % der Leserinnen und Leser spenden nicht.</strong>
+					</li>
```
The list bullets show up as black in firefox, I fear you have to add [custom styling](https://www.w3.org/Style/Examples/007/color-bullets.en.html)

Sperling-0 committed:
Prepare campaign for C23_WMDE_Desktop_DE_24

- Based on the VAR of test 22
- Add direct redirect feature

Ticket: https://phabricator.wikimedia.org/T353637

Abban committed:
Add currentDonationSum to dynamic text

As part of the urgency text we need to add a formatted version
of the current donation sum to the content.

Abban committed:
Prepare campaign for C23_WMDE_Desktop_DE_25

Based on the VAR of test 24

Ticket: https://phabricator.wikimedia.org/T353639

Abban committed:
Implement VAR for C23_WMDE_Desktop_DE_25

VAR has a collapsable banner

Ticket: https://phabricator.wikimedia.org/T353639

Sperling-0 committed:
Archive C23_WMDE_Desktop_DE_24

Ticket: https://phabricator.wikimedia.org/T353639

gbirke commented:
This is broken (my fault during a rebase/cherry pick), so we'll recreate it
